.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: required
+
+services:
+  - docker
+
+language: java
+jdk:
+  - oraclejdk8
+
+before_install:
+  - sed -i.bak -e 's|https://nexus.codehaus.org/snapshots/|https://oss.sonatype.org/content/repositories/codehaus-snapshots/|g' ~/.m2/settings.xml
+  # fire up a IBM MQ container to run the integration tests against
+  - docker-compose -f src/etc/docker/v8/docker-compose.yml up -d
+  # grab the needed proprietary libs from the MQ container and install locally
+  - docker cp local-mq8:/opt/mqm/java/lib/com.ibm.mq.allclient.jar .
+  - docker cp local-mq8:/opt/mqm/java/lib/com.ibm.mq.pcf.jar .
+  - mvn install:install-file -Dfile=com.ibm.mq.allclient.jar -DgroupId=com.ibm -DartifactId=com.ibm.mq.allclient -Dversion=8 -Dpackaging=jar
+  - mvn install:install-file -Dfile=com.ibm.mq.pcf.jar -DgroupId=com.ibm -DartifactId=com.ibm.mq.pcf -Dversion=8 -Dpackaging=jar
```

Comment:
The below local build section must be removed (it references the forked repo) once [spring-cloud-stream-binder-jms](https://github.com/spring-cloud/spring-cloud-stream-binder-jms) follows the usual Spring project build process. I.e. `spring-cloud-stream-binder-jms` artifacts are available as `BUILD-SNAPSHOT`'s.